shared/chat/conversation/messages/unfurl/generic/index.js

```diff
@@ -16,28 +16,51 @@ export type Props = {
   showImageOnSide: boolean,
 }
 
-class UnfurlGeneric extends React.PureComponent<Props> {
+type State = {
+  showClose: boolean,
+}
+
+class UnfurlGeneric extends React.Component<Props, State> {
+  state = {showClose: false}
```

Comment:
would prefer using css for this, esp. since this is effectively desktop only. lets discuss

Reply:
Done!